config/packages/fos_elastica.yaml

```diff
@@ -139,6 +139,10 @@ fos_elastica:
                             properties:
                                 id:
                                     type: 'keyword'
+                        sortingDateForDisplay:
```

Comment:
Couldn't think of a better name for this variable, let me know if you guys think of something, will change it

Reply:
Maybe it's time to bring ApprovalDate to the Dataset entity?

Reply:
What do you mean by ApprovalDate?

Reply:
I just wanted to add "some" date up to the dataset level, but it doesn't seem pratical.